_posts/2021-11-09-solidity-0.8.10-release-announcement.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+published: true
+title: 'Solidity 0.8.10 Release Announcement'
+date: '2021-11-09'
+author: Solidity Team
+category: Releases
+---
+
+[Solidity v0.8.10](https://github.com/ethereum/solidity/releases/tag/v0.8.10) can now report contract invariants and reentrancy properties through the SMT checker. It also contains some new optimizations with regards to external function calls and enabled the new EVM code generator for pure Yul mode.
+
+## Notable New Features
+
+### New EVM Code Transform for compilation via Yul IR
+
+We switch the experimental compilation pipeline via Yul (with enabled optimizer) to a new code generation backend to generate EVM assembly from Yul code.
```

Comment:
Can we reorder this sentence somehow so that it maybe starts with "new EVM code generation backend" - it's a bit difficult to parse like this IMO.

Reply:
Pushed another version - is that better?